code/__DEFINES/misc.dm

```diff
@@ -456,3 +456,7 @@ GLOBAL_LIST_INIT(ghost_others_options, list(GHOST_OTHERS_SIMPLE, GHOST_OTHERS_DE
 #define ROUND_END_NOT_DELAYED 0
 #define ROUND_END_DELAYED 1
 #define ROUND_END_TGS 2
+
+/// A null statement to guard against EmptyBlock lint without necessitating the use of pass()
+/// Used to avoid proc-call overhead. But use sparingly. Probably pointless in most places.
+#define EMPTY_BLOCK_GUARD ;
```

Comment:
If you want to micro performance hits; use this instead of pass(). If you disable the empty block lint entirely per my other comment then this isn't required.

Reply:
True, true. I honestly wasn't sure what circumstances pass() would be used over this define, so I just used pass() for everything. Will swap those out just for the sake of it.

Reply:
Per the other comment if you disable the linter then you don't even need the empty block guard since you won't be linting against it